native/cocos/audio/android/AudioMixerController.h

```diff
@@ -44,12 +49,17 @@ class AudioMixerController {
         size_t size;
     };
 
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
     AudioMixerController(int bufferSizeInFrames, int sampleRate, int channelCount);
+    bool init();
+#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+    AudioMixerController(int sampleRate, int channelCount);
+    void updateBufferSize(int bufferSize);
+    bool init(int bufferSizeInFrames);
```

Comment:
Could not put the logic of initializing `bufferSizeInFrames` to the constructor?

Reply:
![Uploading image.png…]()